scan.py

```diff
@@ -146,78 +139,51 @@ def set_av_tags(s3_client, s3_object, scan_result, scan_signature, timestamp):
     new_tags.append({"Key": AV_STATUS_METADATA, "Value": scan_result})
     new_tags.append({"Key": AV_TIMESTAMP_METADATA, "Value": timestamp})
     s3_client.put_object_tagging(
-        Bucket=s3_object.bucket_name, Key=s3_object.key, Tagging={"TagSet": new_tags}
+        Bucket=s3_object.bucket_name,
+        Key=s3_object.key,
+        Tagging={"TagSet": new_tags}
     )
 
 
-def sns_start_scan(sns_client, s3_object, scan_start_sns_arn, timestamp):
-    message = {
-        "bucket": s3_object.bucket_name,
-        "key": s3_object.key,
-        "version": s3_object.version_id,
-        AV_SCAN_START_METADATA: True,
-        AV_TIMESTAMP_METADATA: timestamp,
-    }
-    sns_client.publish(
-        TargetArn=scan_start_sns_arn,
-        Message=json.dumps({"default": json.dumps(message)}),
-        MessageStructure="json",
-    )
+def callback_server(s3_client, s3_object, event, scan_result, context):
```

Comment:
Note to self: get rid of this. 
Or make it work production first. Something like if domain = localhost search for CALLBACK_DOMAIN on env and use it

Reply:
Well, actually the callback_url should contain the full path with everything you need (even when it's from local dev)